packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -80,6 +80,15 @@ export const SECRETS_MANAGER_PARSE_OWNED_SECRET_NAME = '@aws-cdk/aws-secretsmana
  */
 export const KMS_DEFAULT_KEY_POLICIES = '@aws-cdk/aws-kms:defaultKeyPolicies';
 
+/**
+ * Change the old 's3:PutObject*' permission to 's3:PutObject',
+ * as the former one includes 's3:PutObjectAcl',
+ * which allows changing the visibility of an object written to the Bucket.
+ * Use a feature flag to make sure existing customers who might be relying
+ * on the overly-broad permissions are not broken.
+ */
+export const S3_BUCKET_GRANT_WRITE_ACL = '@aws-cdk/aws-s3:BucketGrantWriteAcl';
```

Comment:
From the name this looks like it 'grants write acl', but in reality it seems to 'SKIP grants write acl'.

Can we name it something more clear like `LimitGrantWrite` ? (I'm willing to forego the word "Bucket" since we already scope this to S3).

Reply:
Agreed. Changed to `S3_GRANT_WRITE_WITHOUT_ACL` / `'@aws-cdk/aws-s3:grantWriteWithoutAcl'`, let me know if this is better!